### PR TITLE
fix broken titles

### LINKED ISSUE #222: titles are broken!!!@@@@1111
on most pages

### PR DESCRIPTION
**Pull request by danrschlosser**
Fixes #222

### CODE REVIEW

File: app/templates/events/archive.html [hunk 1]
```diff
@@ -18,13 +16,13 @@
     <div class="content">
         <div class="pagination">
             <div class="previous">
```
Comment (thebrianzeng) [inline app/templates/events/archive.html]: why is this empty

**Comment by danrschlosser** [comment]
screen shot 2015-08-09 at 3 48 36 pm


@thebrianzeng

**Comment by thebrianzeng** [comment]
:+1: good work

**Comment by danrschlosser** [comment]
thx

**Comment by thebrianzeng** [comment]
although gotta test on all browsers

**Comment by danrschlosser** [comment]
:person_frowning:

**Pull request merged**